action/protocol/execution/testdata-istanbul/iip15-manager.sol

```diff
@@ -0,0 +1,56 @@
+//SPDX-License-Identifier: UNLICENSED
+
+pragma solidity ^0.8.0;
+
+import "https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/access/Ownable.sol";
+
+contract IIP15Manager is Ownable {
+
+    struct ContractInfo {
+        address recipient;
+        bool isRegistered;
```

Comment:
This field is unnecessary and you can use recipient is address(0) to check whether registered.